stacker/providers/aws/interactive.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+
+import yaml
+
+from ... import exceptions
+from .default import (
+    Provider as BaseProvider,
+    retry_on_throttling,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def get_change_set_name():
+    """Return a valid Change Set Name.
+
+    The name has to satisfy the following regex:
+        [a-zA-Z][-a-zA-Z0-9]*
+
+    And must be unique across all change sets.
+
+    """
+    return 'change-set-{}'.format(int(time.time()))
+
+
+class Provider(BaseProvider):
+    """AWS Cloudformation Change Set Provider"""
+
+    def _wait_till_change_set_complete(self, change_set_id):
+        response = retry_on_throttling(
+            self.cloudformation.describe_change_set,
+            kwargs={
+                'ChangeSetName': change_set_id,
+            },
+        )
+        if response["Status"] not in ("FAILED", "CREATE_COMPLETE"):
+            time.sleep(2)
+            return self._wait_till_change_set_complete(change_set_id)
```

Comment:
yea that also works. this can technically have multiple pages which i'm not handling yet.